### DISCUSSION TITLE
Is adding `force-dynamic` supposed to prevent fast refresh?

### DISCUSSION THREAD

**Discussion by YCMitch** [vercel/next.js#51810] [App Router]
I'm not sure whether to log this as a bug or not, as perhaps that's some kind of intended behaviour. 

Adding `export const dynamic = "force-dynamic"` to my `page.jsx` results in _any_ code change in this file triggering a full refresh. If I remove it, this stops happening. 

Is that deliberate?

**Comment by corned-beefhash** [comment]
wondering the same thing
